addons/binding/org.openhab.binding.valloxmv/ESH-INF/thing/thing-types.xml

```diff
@@ -172,6 +174,21 @@
 		<state pattern="%d h" readOnly="true" />
 	</channel-type>
 
+	<channel-type id="filterchangeddate" advanced="true">
+		<item-type>DateTime</item-type>
+		<label>Last filter change</label>
+		<description>Date filter was changed last time</description>
+		<category>Date</category>
```

Comment:
The "Date" category doesn't exist so UIs would not know how to handle it. There is a "Time" category. See [categories](https://www.eclipse.org/smarthome/documentation/concepts/categories.html) on how they are used and what categories are available.

Reply:
Took the categorie out

(honestly just used an example taken from ntp:

```
	<channel-type id="dateTime-channel">
		<item-type>DateTime</item-type>
		<label>Date</label>
		<description>NTP refreshed date &amp; time</description>
		<category>Date</category>
		<state readOnly="true" />
	</channel-type>
```
)